CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Traffic Portal no longer uses `ruby compass` to compile sass and now uses `dart-sass`.
 - Changed Invalidation Jobs throughout (TO, TP, T3C, etc.) to account for the ability to do both REFRESH and REFETCH requests for resources.
 - The `admin` Role is now always guaranteed to exist, and can't be deleted or modified.
+- Updated `t3c-apply` to reduce mutable state in `TrafficOpsReq` struct.
```

Comment:
This is an internal tech-debt thing. It doesn't really need to be in the changelog, since it doesn't change anything for users in any way